### ISSUE TITLE
Windows 11 touch keyboard hides service window contents

### ISSUE THREAD

**Issue by halms** [getferdi/ferdi#2205]
**Bug description:**
When used on a Surface device without a keyboard attached, the onscreen keyboard opens once the cursor focuses an input area (as it should!).
The issue is that the onscreen keyboard hides large parts of the service, especially the text input area of, e.g., WhatsApp.
This is *not* service-specific, it happens in WhatsApp, Telegram and many others.

This happens on my Surface Pro 7 with Windows 11, but I think it generally applies to touch screen devices also under Windows 10.

**Steps to reproduce:**
1. Use a touch device without attached keyboard (e.g., Surface Pro).
2. Open a Service  in Ferdi
3. Tap on any text input area, e.g., composing a new message in WhatsApp
4. Keyboard pops up, but window/content does not resize. Therefore Keyboard hides text input area, rendering WhatsApp any many other services mostly unusable.

**Expected behavior**
The Ferdi window or content in the Ferdi windows should resize to move out of the way from the keyboard.

Basically all other apps handle this correctly, specifically also other Electron apps, such as Signal Desktop or VS Code.

**Screenshots**
![Screenshot 2021-11-10 012227](https://user-images.githubusercontent.com/7513146/141028379-c716149d-7139-4bf2-a40c-fdbc35cfd735.png)

**Environment:**

- Operating System: Windows 11 (also applies to Windows 10)
- Ferdi Version: 5.6.3
- Server: Ferdi, but not relevant

- Debug information: Should not be relevant.
